### PR TITLE
Added an optional `height` and overflow scrollbar for the Markdown Component

### PR DESCRIPTION
**Pull request by ShruAgarwal**
## Description

Adds an option to enable the `height` parameter `(type: str)` for a scrollable markdown. This can be used for controlling the display size of this component.

Please run `demo/markdown_example` to see it in action.

_I kindly request you @abidlabs to please review this change and let me know your feedback!_ 🙂 

Closes: #7455

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/markdown/shared/Markdown.svelte [hunk 1]
```diff
@@ -30,6 +31,7 @@
 	data-testid="markdown"
 	dir={rtl ? "rtl" : "ltr"}
 	use:copy
+	style="max-height: {height}; overflow-y: auto;"
```
Comment (freddyaboulton) [inline js/markdown/shared/Markdown.svelte:34]: I feel like this style should only be applied if the height is not null

Comment (ShruAgarwal) [reply js/markdown/shared/Markdown.svelte]: I have made suggested changes and would request you to review and share your thoughts @freddyaboulton !

**Review by abidlabs — APPROVED** [review]
Looks great, thanks for this nice PR @ShruAgarwal! I made some tweaks for consistency with other components e.g. to make the `height` parameter also be able to accept an int.

**Comment by ShruAgarwal** [comment]
> Looks great, thanks for this nice PR @ShruAgarwal! I made some tweaks for consistency with other components e.g. to make the `height` parameter also be able to accept an int.

Thanks a lot, @abidlabs ! 🤗😀

**Pull request merged**